raiden-ts/src/config.ts

```diff
@@ -0,0 +1,32 @@
+import * as t from 'io-ts';
+
+/**
+ * An Raiden configuration object to overwrite the Raiden default configuration.
+ * 
+ * @matrixServer Matrix server to initliaze the [[Raiden]] instance with.
+ * The matrix server can't be changed later on.
+ */
+export const PartialRaidenConfig = t.partial({
```

Comment:
This can go inside `export const RaidenConfig`, it isn't used explicitly anywhere, so it should be inlined there.